src/dcos_e2e/backends/_docker/__init__.py

```diff
@@ -462,7 +469,7 @@ def __init__(  # pylint: disable=super-init-not-called,too-many-statements
             'ssh_port':
             22,
             'ssh_user':
-            'root',
+            cluster_backend.default_ssh_user,
```

Comment:
I love this because it makes it easier for us to move this config creation into `Cluster`.